packages/core/src/oidc/adapter.ts

```diff
@@ -32,7 +33,13 @@ export default function postgresAdapter(modelName: string): ReturnType<AdapterFa
   }
 
   return {
-    upsert: async (id, payload, expiresIn) => upsertInstance(modelName, id, payload, expiresIn),
+    upsert: async (id, payload, expiresIn) =>
+      upsertInstance({
+        modelName,
+        id,
+        payload,
+        expiresAt: dayjs().add(expiresIn, 'second').unix(),
```

Comment:
can we use standard `Date` for pg's `timestamp` column? 

```js
dayjs().add(expiresIn, 'second').toDate();
```

Reply:
concern about the time zone issue. unix timestamp will be more precise.

do you know some best practices on db time storage? would like to change if storing `Date` object has more advantages comparing to `.unix()`.

Reply:
a `Date` instance (actually will be parsed into ISO string or someting like that) has timezone info, while unix has not have. 
- Date: code runtime machine timezone.
- unix: database machine timezone.

If we have 2 replicas in different timezone and write to a single database by unix, it may cause problem.

Reply:
sorry, the problem above is about `Date VS YYYY-MM-DD HH:mm:ss`, not unix.

as for timezone issue,`Date` and `unix` will do the same.

Reply:
But Date do have much info, such as milisecond, though may be useless for now, my point is that Date can do more than unix

Reply:
agreed with millisecond - we can align it across the db.

shall we discuss this in slack and i'll open another PR for time storage aligning?

Reply:
sure